apps/system/js/bluetooth.js

```diff
@@ -30,18 +30,6 @@ var Bluetooth = {
     }
   },
 
-  getCurrentProfiles: function bt_getCurrentProfiles() {
```

Comment:
Good catch! I find out that `smart_system` of `tv_apps` has the same function.
https://github.com/mozilla-b2g/gaia/blob/master/tv_apps/smart-system/js/bluetooth.js#L34-L44